app/test/e2e/utils/helpers.js

```diff
@@ -134,6 +134,7 @@ const mockSubscriptionFindForUser = (userId, idsList = []) => {
 const mockSubscriptionFindAll = (ids = [], overrideArray = {}) => {
     nock(process.env.CT_URL)
         .get(`/v1/subscriptions/find-all`)
+        .query(() => true)
```

Comment:
this is evil - make the mocks match the actual requests.

Reply:
Validating if the correct params are being provided!